content/blog/2022/02/multiplatform-kubewarden.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Multiplatform Kubewarden
+authors:
+    - Rafael Fernández López
+date: 2022-02-01
+---
+
+The Kubewarden team is glad to announce that in the spirit of helping
+Policy Authors and Cluster Administrators, the project is now
+officially multiplatform.
+
+The list of supported platforms as of now are:
+
+- Policy Server, as a [container image](https://github.com/kubewarden/policy-server/pkgs/container/policy-server):
+  - `linux/amd64` (with the `musl` libc)
+  - `linux/arm64` (with the `musl` libc)
+
+- kwctl, as a [standalone binary](https://github.com/kubewarden/kwctl/releases):
+  - `darwin` (`x86_64`)
+  - `linux` (`aarch64`, with the `musl` libc)
+  - `linux` (`x86_64`, with the `musl` libc)
+  - `windows` (`x86_64`, with `MSVC`)
+
+We have prioritized the usage of the same dependencies and toolchain
+on platforms where we were able to do so.
+
+## Looking for platform convergence
+
+Initially, we were using `glibc` and `openssl` as dependencies in our
+project. However, these two libraries are somewhat special and
+sometimes problematic to converge into other platforms.
+
+Here is a short resume of the lessons learned, sometimes the hard way,
+for both libraries.
+
+### The libc
+
+`glibc` only applies to `linux` builds, so in principle is not what
+keeps us from multi platform convergence. However, we thought that
+producing a final static binary for our users in all platforms is
+interesting; specially for a CLI tool like `kwctl`, that our users
+will interact with directly in their environment.
+
+However, compiling a static binary with `glibc` is tricky and can
+become problematic.
+
+The [`musl`](https://musl.libc.org/) libc is providing us with this
+safe, stable and straightforward static linking for our binaries
+targeting linux.
```

Comment:
```suggestion
In the early stage of development, the Kubewarden team decided to use the [`musl libc`](https://musl.libc.org/) library.
This allowed us to avoid potential difficulties with `glibc` down the road (i.e. Cross-compilation time or making a full static binary).

The team also thinks that producing a final static binary for all platforms is in our users own interests; specially for a CLI tool like `kwctl`, which interacts directly with our users' environment.
```